pkg/chain/ethereum/beacon/gen/cmd/cmd.go

```diff
@@ -2,15 +2,19 @@ package cmd
 
 import (
 	"github.com/spf13/cobra"
+
+	"github.com/keep-network/keep-core/pkg/chain/ethereum"
 )
 
-// Command is the exported list of generated commands that can be
+// ModuleCommand is the exported list of generated commands that can be
 // installed on a CLI app. Generated contract command files set up init
 // functions that add the contract's command and subcommands to this global
 // variable, and any top-level command that wishes to include these commands can
 // reference this variable and expect it to contain all generated contract
 // commands.
-var Command = &cobra.Command{
-	Use:   "beacon",
-	Short: `Provides access to Keep Random Beacon contracts.`,
+var ModuleCommand = ethereum.Command{
+	Command: cobra.Command{
+		Use:   "beacon",
+		Short: `Provides access to Keep Random Beacon contracts.`,
```

Comment:
I'm not 100% sure, but didn't we stop using "Keep" in favor of "T"? Keep Network -> Threshold Network? Same comment for other places where we use "Keep"

Reply:
I'm not sure we fully transitioned. I'd keep it as is and if we decide to rename we would do this in a bulk PR.